spec/unit/puppet_pal_2pec.rb

```diff
@@ -112,18 +104,6 @@
             expect { ctx.with_script_compiler {|c| c.evaluate_string('$a') }}.to raise_error(/Unknown variable: 'a'/)
           end
         end
-
-        it 'instantiates definitions in the given code string' do
```

Comment:
This test shoud remain, but be changed to not run a plan, but some other definition - like a function.
Simply change to "function run_me()" and use "call_function". It is not testing plan per se.

Reply:
Done.